eclipse/sw360
# feat(ci): Run Client Integration Test for rest api on DB

> Run Client Integration Test for rest api on DB.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> Check the Gitlab action logs and validate
> Test new endpoint 
>    - Delete License - http://localhost:8080/resource/docs/api-guide.html#resources-license-delete
>    - Delete Project

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
